docs/Installation.md

```diff
@@ -16,3 +16,14 @@ Pairing PRs is handled automatically by running this command from a root directo
 ```bash
    make ci/test/e2e
 ```
+
+## Install Red Hat App Studio in e2e mode in kcp
+
+The Red Hat App Studio installation scripts support kcp. To install App studio in e2e mode for kcp we need to execute the following script:
+
+```bash
+   export OFFLINE_TOKEN=<the_token_goes_here>
```

Comment:
should there be an info about what kind of token is needed?

Reply:
agree ;-)